nautobot/extras/api/views.py

```diff
@@ -562,7 +574,9 @@ def _run_job(request, job_model, legacy_response=False):
 
     # Try to create a ScheduledJob, or...
     if schedule_data:
-        schedule = _create_schedule(schedule_data, data, commit, job, job_model, request)
+        schedule = _create_schedule(
+            schedule_data, job_class.serialize_data(cleaned_data), commit, job, job_model, request
```

Comment:
Does this work OK for jobs that contain ObjectVars (where we pass into the API a PK or similar identifier, but the job should eventually execute with an object instance)?

Reply:
Yes this actually aligns the APIs to send in data the same as what is being done in the UI as well (see: https://github.com/nautobot/nautobot/blob/v1.4.7/nautobot/extras/views.py#L1169)

And this test is testing for that: https://github.com/nautobot/nautobot/pull/2644/files#diff-7a3e1e4d7298b8ac69ba134aa83e397840aec9fda33ad86f8e34a9981256927eR1379-R1382